### ISSUE TITLE
Unresolved import `malachite_q`

### ISSUE THREAD

**Issue by ruescasd** [mhogrefe/malachite#6]
When building:

```
error[E0432]: unresolved import `malachite_q`
 --> C:\Users\ruesc\.cargo\registry\src\github.com-1ecc6299db9ec823\malachite-0.3.0\src\lib.rs:7:9
  |
7 | pub use malachite_q::Rational;
  |         ^^^^^^^^^^^ use of undeclared crate or module `malachite_q`
  |
help: there is a crate or module with a similar name
  |
7 | pub use malachite_nz::Rational;
  |         ~~~~~~~~~~~~
```

Cargo.toml dependency:

```[dependencies.malachite]
version = "0.3.0"
default-features = false
features = [ "naturals_and_integers" ]
```

**Comment by mhogrefe** [comment]
I just released Malachite 0.3.1, which fixes this bug. Please let me know if upgrading to this version doesn't work for you.

**Comment by ruescasd** [comment]
Works for me thanks!
